evap/staff/tests.py

```diff
@@ -85,6 +85,18 @@ def test_sample_user_xls(self):
 
         self.assertEqual(UserProfile.objects.count(), original_user_count + 2)
 
+    def test_sample_user_xls_import_participants(self):
+        course = mommy.make(Course, pk=1)
+        page = self.app.get("/staff/semester/1/course/1/importparticipants", user='user')
+
+        original_participant_count = course.participants.count()
+
+        form = lastform(page)
+        form["excel_file"] = (os.path.join(settings.BASE_DIR, "static", "sample_user.xls"),)
+        form.submit(name="operation", value="import")
+
+        self.assertEqual(course.participants.count(), original_participant_count + 2)
+
```

Comment:
that would be your opportunity to add an example how to properly test a view ;) you could add a new class somewhere in here and put all the tests in there.